pkg/apis/azure/validation/infrastructure.go

```diff
@@ -16,12 +16,16 @@ package validation
 
 import (
 	apisazure "github.com/gardener/gardener-extension-provider-azure/pkg/apis/azure"
-
 	cidrvalidation "github.com/gardener/gardener/pkg/utils/validation/cidr"
 	apivalidation "k8s.io/apimachinery/pkg/api/validation"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 )
 
+const (
+	natGatewayMinTimeoutValue int32 = 4
```

Comment:
You could specify that this timeout is in minutes. Otherwise it is not clear.

```suggestion
	natGatewayMinTimeoutInMinutes int32 = 4
```